### ISSUE TITLE
Plotly does not build during `gatsby build`

### ISSUE THREAD

**Issue by hulloanson** [bikespace/bikespace#147]
To reproduce:

```bash
git checkout develop
# at root of repo
make build-frontend
```

Looks like plotly does not play well with SSR.

Relevant error log:

```plaintext
 ERROR  UNKNOWN

Truncated page data information for the failed page "/dashboard/": {
  "errors": {},
  "path": "/dashboard/",
  "slicesMap": {},
  "pageContext": {}
}

failed Building static HTML for pages - 5.157s

 ERROR #95313  HTML.COMPILATION

Building static HTML failed for path "/dashboard/"

See our docs page for more info on this error: https://gatsby.dev/debug-html


  14 |  else
  15 |          root["Plotly"] = factory();
> 16 | })(self, function() {
     |  ^
  17 | return /******/ (function() { // webpackBootstrap
  18 | /******/         var __webpack_modules__ = ({
  19 |


  WebpackError: ReferenceError: self is not defined
  
  - plotly.js:16 
    [bikespace_frontend]/[plotly.js]/dist/plotly.js:16:2
  
  - bootstrap:19 
    bikespace_frontend/webpack/bootstrap:19:1
  
  - react-plotly.js:10 
    [bikespace_frontend]/[react-plotly.js]/react-plotly.js:10:38
  
  - bootstrap:19 
    bikespace_frontend/webpack/bootstrap:19:1
  
  - index.js:972 
    [bikespace_frontend]/[node-fetch]/lib/index.js:972:1
  
  - bootstrap:19 
    bikespace_frontend/webpack/bootstrap:19:1
  
  - static-entry.js:224 
    bikespace_frontend/.cache/static-entry.js:224:27
  
  - index.js:387 
    [bikespace_frontend]/[node-fetch]/lib/index.js:387:1
```

**Comment by xinweny** [comment]
I implemented a fix using `react-loadable`, feel free to try it out!

**Comment by hulloanson** [comment]
Closed by #148
